playground/backend/internal/cache/cache.go

```diff
@@ -55,6 +55,9 @@ const (
 
 	// LogsIndex is the index of the start of the log
 	LogsIndex SubKey = "LOGS_INDEX"
+
+	// ExamplesCatalog is catalog of examples available in Playground
+	ExamplesCatalog SubKey = "EXAMPLES_CATALOG"
```

Comment:
Instead, make workarounds with subkey without pipeline id. Can we extend the Cache interface and implementation with  SetCatalog, GetKatalog?

Reply:
You are right, changed